stan-models/AR1.R

```diff
@@ -0,0 +1,158 @@
+library(dplyr)
+library(tidyr)
+library(rstan)
+devtools::load_all()
+
+start_yr <- 1982
+end_yr <- 2013
+min_num_yrs <- 25
+last_training_year <- 2003
+richness_w_env <- get_richness_ts_env_data(start_yr, end_yr, min_num_yrs) %>%
```

Comment:
Not sure how it affects the model yet, but `get_richness_ts_env_data()` is not enforcing `min_num_years`, there are only ~1000 sites that have >25 years of data.